src/pyglimer/waveform/download.py

```diff
@@ -813,7 +817,7 @@ def downloadwav(
     # # Create handler to the log
     if log_fh is None:
         fh = logging.FileHandler(os.path.join('logs', 'download.log'))
-        fh.setLevel(logging.INFO)
+        fh.setLevel(logging.info)
```

Comment:
this is not correct. ``logging.info`` is a function, whereas ``logging.INFO`` is the actual logging level.